examples/misc/bin/try_dart/classes.dart

```diff
@@ -0,0 +1,41 @@
+// ignore_for_file: annotate_overrides, sort_constructors_first, type_annotate_public_apis
+// Abstract classes can't be instantiated.
+// ignore: one_member_abstracts
+abstract class Item {
+  use();
+}
+
+// Classes can implement other classes.
+class Chest<T> implements Item {
+  List<T> contents;
+
+  // Constructors can assign arguments to instance variables using `this`.
+  Chest(this.contents);
+
+  use() => print("$this has ${contents.length} items.");
+}
+
+class Sword implements Item {
+  int damage = 5;
+
+  use() => print("$this dealt $damage damage.");
+}
+
+// Classes can extend other classes.
+class DiamondSword extends Sword {
+  int damage = 50; // ignore: overridden_fields
```

Comment:
Let's instead change the `damage` field to a getter to make this warning go away